src/libraries/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Net
     public partial class AuthenticationManager
     {
         internal AuthenticationManager() { }
-        public static System.Net.ICredentialPolicy CredentialPolicy { get { throw null; } set { } }
+        public static System.Net.ICredentialPolicy? CredentialPolicy { get { throw null; } set { } }
         public static System.Collections.Specialized.StringDictionary CustomTargetNameDictionary { get { throw null; } }
         public static System.Collections.IEnumerator RegisteredModules { get { throw null; } }
         public static System.Net.Authorization Authenticate(string challenge, System.Net.WebRequest request, System.Net.ICredentials credentials) { throw null; }
```

Comment:
Doc says this and below method returns null https://docs.microsoft.com/en-us/dotnet/api/system.net.authenticationmanager.authenticate?view=netframework-4.8#returns

Reply:
Thanks. Fixed.